<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraView</class>
 <widget class="QDialog" name="CameraView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_detection">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>开始红灯检测</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,6">
        <item>
         <widget class="QLabel" name="label_videoIP">
          <property name="text">
           <string>视频流IP：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_videoIP"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,6">
        <item>
         <widget class="QLabel" name="label_videoPort">
          <property name="text">
           <string>视频流端口：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_videoPort"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="4,6">
        <item>
         <widget class="QLabel" name="label_command">
          <property name="text">
           <string>特殊命令：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_videocommand"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_openVideo">
      <property name="text">
       <string>打开视频流</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_openVideo</sender>
   <signal>clicked()</signal>
   <receiver>CameraView</receiver>
   <slot>openVideoStream()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_detection</sender>
   <signal>clicked()</signal>
   <receiver>CameraView</receiver>
   <slot>trafficDetection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openVideoStream()</slot>
  <slot>trafficDetection()</slot>
 </slots>
</ui>
